.gitignore

```diff
@@ -0,0 +1,8 @@
+**/__pycache__
```

Comment:
The `.gitignore` file does not include a trailing slash for `__pycache__`, which may result in unintended files being ignored. Add a trailing slash to ensure only directories are ignored.